README.md

```diff
@@ -107,6 +107,31 @@ Installing the library is as easy as:
 ember install ember-simple-auth
 ```
 
+### TypeScript setup
+1. Include `ember-simple-auth` typescript defs in the `tsconfig.json` under `paths` like this:
```

Comment:
Can we make a note that this may not be required depending on your TS configuration. I am not sure that this is strictly required in every situation. If you are positive it is needed then please disregard.